### ISSUE TITLE
Attachment Enhancements

### ISSUE THREAD

**Issue by Mr-Anonymous** [getk2/k2-v3-dev-build#80]
Couple of tweaks are pending on Content Attachment tab:

1) When you click 'Upload' to add a file from your computer, it is better to show the selected filename here. Right now, I see that although a file is selected from my computer, since I dont see the selected filename shown here, I was thinking the file was not selected. Only after clicking 'save' I understood that the file was indeed selected but it wasnt showing. (I can see the filename as a hidden input in the source)

2) The loading gif that shows when saving the item after adding the attachment doesnt seem to be an animated gif. (this is just a minor thing.. not a biggie)

3) Right now, the 'Remove' button for attachment row (for the already attached fles) doesnt show a confirmation before removing. Wouldn't a confirmation be useful so users dont remove it by mistake?
![attachments](https://cloud.githubusercontent.com/assets/8770270/4344633/5a7387ec-409c-11e4-81f4-1d9733b2e221.jpg)

**Comment by paulvanharen** [comment]
Duplicate of #28
